### PR TITLE
theme designer: make name editable

### PR DESCRIPTION
**Pull request by emmayjiang**
Makes the name of the theme editable.

image

### CODE REVIEW

**Comment by emmayjiang** [comment]
First step of ramp is no longer #000000

image

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/ad8661e299d2528a2d191469f52b2000a293dd72'>ad8661e299d2528a2d191469f52b2000a293dd72</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by emmayjiang** [comment]
change brand[#] in export to match custom name

File: packages/react-components/theme-designer/src/components/Nav/Nav.tsx [hunk 1]
```diff
@@ -32,17 +34,20 @@ const useStyles = makeStyles({
...
 export const Nav: React.FC<NavProps> = props => {
   const styles = useStyles();
+
+  const underlineId = useId('input-underline');
+  const name = useContextSelector(AppContext, ctx => ctx.name);
+  const setName = useContextSelector(AppContext, ctx => ctx.setName);
+
+  const handleNameChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const newName = e.target.value;
+    if (newName.match('^[a-zA-Z0-9]*$')) {
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/components/Nav/Nav.tsx:46]: don't restrict the name and force the user to know string options, parse it for them

**Review by micahgodbolt — APPROVED** [review]

**Pull request merged**